examples/prefect/02-prefect-cloud.ipynb

```diff
@@ -384,47 +385,47 @@
     "* `n_workers = 3`: use 3 workers\n",
     "* `worker_size =\"xlarge\"`: each worker has 2 CPU cores and 16 GB RAM\n",
     "    - **NOTE**: you can find the full list of sizes with `prefect_saturn.describe_sizes()`\n",
-    "* `autoclose = False`: leave the Dask cluster running after the flow run finishes\n",
-    "* `worker_is_spot = False`: don't use spot instances for workers"
-   ],
-   "metadata": {}
+    "* `worker_is_spot = False`: don't use spot instances for workers\n",
+    "\n",
+    "**NOTE:** dask clusters associated with prefect cloud flows will be autoclosed when the flow run completes."
+   ]
```

Comment:
This may need to change based on the outcome of https://github.com/saturncloud/dask-saturn/pull/60#discussion_r703882777
Right now it is correct